### PR TITLE
Document AddressFormat class

### PR DESCRIPTION
**Pull request by firstred**
| Questions | Answers |
| --- | --- |
| Branch? | `develop` |
| Description? | Upgrading the PHPDoc comments of this class, using the latest PS standard. Also getting the Franglais out of the comments, getting rid of things my Symfony code sniffer nags about, etc. It might have some additional `todo` tags, depending on whether the code was comprehensible or not. |
| Type? | improvement |
| Category? | CO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | N/A |
| How to test? | N/A |

### CODE REVIEW

File: classes/AddressFormat.php [hunk 1]
```diff
@@ -49,13 +52,16 @@ class AddressFormatCore extends ObjectModel
         ),
     );
 
+    /** @var array Default required form fields list */
     public static $requireFormFieldsList = array(
```
Comment (mickaelandrieu) [inline classes/AddressFormat.php]: Should be ``$requiredFormFieldList`` but this is a breaking change ...  ping @julienbourdeau

File: classes/AddressFormat.php [hunk 2]
```diff
@@ -284,9 +298,9 @@ protected static function _setOriginalDisplayFormat(&$formattedValueList, $curre
         }
     }
 
-    /*
-    ** Cleaned the layout set by the user
-    */
+    /**
```
Comment (mickaelandrieu) [inline classes/AddressFormat.php]: note this second asterisk is not required in PSR, this is only useful when you want to work with Doctrine and Symfony annotations :)

**Comment by mickaelandrieu** [comment]
I'll probably merge it tomorrow (today ? ^^), after Julien little review on the breaking change I want to introduce :p

File: classes/AddressFormat.php (hunk 2 above)
Comment (firstred) [inline classes/AddressFormat.php]: Yeah, I'm trying to upgrade to Symfony standards. This also works much better with PhpStorm btw.

**Comment by mickaelandrieu** [comment]
Let's merge it this way, if we need to update/improve the variables names we will do it in another one.

Thanks !

**Comment by firstred** [comment]
Yep, that was probably out of the scope of this documentation

Maybe a `TODO` will do for now?

**Comment by mickaelandrieu** [comment]
Don't worry, we will do the update (or not) asap ^^ don't give us too much work !

**Pull request merged**